Client/game_sa/gamesa_init.h

```diff
@@ -104,3 +104,58 @@ void MemOrFast(U ptr, const T value)
 
 bool GetDebugIdEnabled(uint uiDebugId);
 void LogEvent(uint uiDebugId, const char* szType, const char* szContext, const char* szBody, uint uiAddReportLogId = 0);
+
+struct __THISCALL{};
+struct __CDECL{};
+struct __STDCALL{};
+struct __FASTCALL{};
+struct __VECTORCALL{};
```

Comment:
How about using a namespace for these tag structs?

Reply:
Good idea. I think you can then remove the "__" or is it better to leave it?

Reply:
if its in a namespace, yes.

Reply:
Done. For cdecl I had to add a prefix because cdecl and CDECL are defined as macros and the compiler throws errors

Reply:
> Done. For cdecl I had to add a prefix because cdecl and CDECL are defined as macros and the compiler throws errors

In that case maybe `__XYZ` format isnt so bad after all

Reply:
I think so too